pages/dkp/kommander/1.4/clusters/attach-cluster/requirements-for-attaching/index.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: layout.pug
+navigationTitle: Requirements for Attaching an Existing Cluster to Kommander
+title: Requirements for Attaching an Existing Cluster to Kommander
+menuWeight: 3
+excerpt: Requirements for attaching an existing cluster to Kommander
+beta: true
+---
+
+## Basic requirements
+
+You can attach existing clusters to Kommander whether or not those clusters have networking restrictions.
+
+For attaching existing clusters without networking restrictions, the requirements depend on the Kommander 1.X version that you plan to use. The supported Kubernetes versions are between version 1.16.x and 1.20.x with a maximum version of 1.20.5. To attach an existing cluster, the Kommander management cluster must be able to reach the services and the api-server of the target cluster.
+
+For attaching clusters _with_ networking restrictions, the Kubernetes version of the cluster you wish to attach must be greater than or equal to version 1.19.x. For these types of clusters, the Kommander cluster uses a tunnel to access the clusters you plan to attach. Any managed service you want to expose in the Kommander cluster requires the creation of a reverse proxy, or the use of HTTPS_PROXY environment variables that point to the Kommander tunnel proxy server.
+
+When working with clusters that have networking restrictions, you need to run some YAML manifests on the existing cluster, so that Kommander can collect a resulting `kubeconfig` file used to establish the tunnel.
+
+### Projects and Workspaces
+
+Before you attach clusters, you need to create one or more Projects and Workspaces. [Projects][projects] allow you to define one or more clusters as a group to which Kommander pushes a common configuration. [Workspaces][workspaces] give you a logical way to represent your teams and specific configurations. Grouping your existing clusters in Kommander projects and workspaces makes managing their plaform services and resources easier and supports monitoring and logging.
```

Comment:
We don't really have to create a project first. You do need a workspace though (even though here is a default workspace already created upon Kommander deployment, we can say that they need one, because it's true. And, projects reside within workspaces, so I just added that in there.

```suggestion
Before you attach clusters, you need to create one or more Workspaces, and we encourage creating Projects within your Workspaces. [Workspaces][workspaces] give you a logical way to represent your teams and specific configurations. [Projects][projects] allow you to define one or more clusters as a group in a Workspace to which Kommander pushes a common configuration. Grouping your existing clusters in Kommander projects and workspaces make managing their platform services and resources easier and supports monitoring and logging.
```

Reply:
Order of sentences changed as noted. In the last sentence, the verb does need to be plural. The words "projects and workspaces" are the direct object of the prepositional phrase, not the subject of the sentence.